bindata/v4.1.0/config/defaultconfig.yaml

```diff
@@ -88,8 +88,8 @@ authConfig:
 consolePublicURL: ""
 kubeletClientInfo:
   ca: /etc/kubernetes/static-pod-resources/configmaps/kubelet-serving-ca/ca-bundle.crt
-  certFile: /etc/kubernetes/static-pod-resources/secrets/kubelet-client/tls.crt
-  keyFile: /etc/kubernetes/static-pod-resources/secrets/kubelet-client/tls.key
```

Comment:
what does this mean? We never wired the kubelet client cert correctly? How did it work like that?

Reply:
Please check 4.4 and older.

Reply:
it was located in both.  this is just updating us to point at a maintained file.  this looks correct.

Reply:
Thanks @deads2k.

I figured out that this was needed after my first try to install failed at bootstrap because I did not change the wiring.